src/modules/roc_sndio/ibackend.cpp

```diff
@@ -14,5 +14,20 @@ namespace sndio {
 IBackend::~IBackend() {
 }
 
+bool IBackend::add_driver_uniq(core::Array<DriverInfo>& arr, const char* driver_name) {
+    for (size_t n = 0; n < arr.size(); n++) {
+        if (strcmp(driver_name, arr[n].name) == 0) {
```

Comment:
strcmp() will crash if driver_name is NULL. I suggest to handle this case more gracefully, e.g. insert a panic:

```
bool IBackend::add_driver_uniq(core::Array<DriverInfo>& arr, const char* driver_name) {
    roc_panic_if(driver_name == NULL);
    ...
```

or some other way.